storage-bigtable/src/lib.rs

```diff
@@ -70,6 +70,10 @@ fn slot_to_key(slot: Slot) -> String {
     format!("{:016x}", slot)
 }
 
+fn slot_to_blocks_key(slot: Slot) -> String {
```

Comment:
intention here is that DRY.

hopefully, `slot_to_tx_by_addr_key` and etc can be defined as well. (the `!slot` bit-or is quite easy to forget to do properly at call site...)

Reply:
done 429bbcb152b71f24f75c2dcb1534d75fb21cadc1